tasks/build_tags.py

```diff
@@ -78,7 +75,6 @@
     "orchestrator",
     "otlp",
     "podman",
-    "process",
```

Comment:
If you can rebase/merge and remove the "process" from the system-probe list as well, thanks !

Reply:
Oh yeah good catch !
I'll finish my checks for this branch later today and open it if everything is fine by the way